packages/instantsearch.js/src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -937,10 +937,10 @@ describe('insights', () => {
             createInsightsMiddleware({ insightsClient })
           );
 
-          expect(getUserToken()).toEqual('abc');
+          expect(getUserToken()).not.toEqual('abc');
```

Comment:
It going to be a generated anon token with a uuid, so hard to test the exact string value otherwise. Is there any other method I can use to assert that?

Reply:
```suggestion
          expect(getUserToken).toEqual(expect.stringMatching(/^anonymous-/));
```